content/tutorials/themes/theme-objects/breadcrumb/index.md

```diff
@@ -36,19 +36,23 @@ The starting level, separator and styliong are configurable.
 <object id="dnnBREADCRUMB" codetype="dotnetnuke/server" codebase="BREADCRUMB">
 ```
 
-
-| Attribute | Description                              | Default    | Values                                                                                                                                                                                       | Version    |
-|-----------|------------------------------------------|------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|
-| Separator | The separator between breadcrumb links\. | >          | html / text / image                                                                                                                                                                          | 01\.00\.00 |
-| CssClass  | CSS Class of the SkinObject              | SkinObject | A valid CSS Class                                                                                                                                                                            | 01\.00\.00 |
-| RootLevel | The root level of the breadcrumb links\. | 1          | Valid values include:<br> \-1  \-  show word “Root” and then all breadcrumb tabs<br> 0  \-  show all breadcrumb tabs<br> n \(Integer > 0\)  \-  skip n breadcrumb tabs before displaying<br> | 01\.00\.00 |
-| UseTitle  | Use the PageTitle instead of PageName    | False      | True / False                                                                                                                                                                                 | 01\.00\.00 |
-
-
+| Attribute | Description | Default | Values | DNN Version |
+| --- | --- | --- | --- | --- |
+| Separator | The separator between breadcrumb links. | > | html / text / image | 01.00.00 |
+| CssClass | CSS Class of the SkinObject | SkinObject | A valid CSS Class | 01.00.00 |
+| RootLevel | The root level of the breadcrumb links. | 1 | Valid values include:<br/>-1 - show word “Root” and then all breadcrumb tabs<br/>0 - show all breadcrumb tabs<br/>n (Integer > 0) - skip n breadcrumb tabs before displaying | 01.00.00 |
```

Comment:
I didn't have time to test to see if this translates and renders correctly via local build.  Did you?

Reply:
No I didn't TBH

Reply:
I did try a markup visualizer and there it worked fine..

Reply:
Markdown visualizers can be okay, but keep in mind this is [DocFx flavored markdown](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html?tabs=tabid-1%2Ctabid-a).  So, the best way to ensure it is working is to just run `docfx serve` locally.

Reply:
@Timo-Breumelhof it looks like this rendered okay, so I'll mark it as resolved.